Python/chapter01/1.1 - Is Unique/miguel_1.1_sol.py

```diff
@@ -0,0 +1,60 @@
+"""
+Python version 3.7.0
+1.1
+Implement an algorithm to determine if a string has all unique characters.
+What if you cannot use additional data structures?
+
+Given:		Expect:
+tacos		true
+swag		true
+bobby		false
+california	false
+orbit		true
+e		    true
+"""
+
+
+def is_unique(input_str):
+    freqs = {}
+    for c in input_str:
+        if c not in freqs:
+            freqs[c] = 1
+        else:
+            return False
+    return True
+
+
+def is_unique_no_additional_data_structures(input_str):
+    for i in range(0, len(input_str)):
+        for j in range(i+1, len(input_str)):
```

Comment:
here you just want to check if the character is contained within a substring:

```suggestion
        if c in input_str[i+1:]:
            return False
```